content/lwrp_to_custom_resources.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Migrating from HWRPs to Custom Resources"
+gh_repo = "chef-web-docs"
+
+product = ["client", "workstation"]
+
+[menu]
+  [menu.infra]
+    title = "Migrating from HWRPs"
+    identifier = "chef_infra/resources/custom_resources/hwrp"
+    parent = "chef_infra/resources/custom_resources"
+    weight = 120
++++
+
+TODO: Is heavy weight because doing it directly uses all of the class
+## Overview
+
+It is no longer recommended to write resources in the __Heavy Weight Resource Provider (HWRP)__ format.
+
+This guide describes how to migrate from an existing HWRP to a Custom Resource.
+
+If you are creating a Custom Resource from scratch please see the [Custom Resource Getting Started Guide](getting-started.md) instead.
+
+## Convert files to Custom Resources Layout
+
+HWRPs consist of two library files: a resource and a provider for that resource.
+
+```text
+|- libraries
+  |- provider_rvm_ruby.rb
+  |- resource_rvm_ruby.rb
+```
```

Comment:
from @tas50 `If there are two library files it's not an LWRP. It's an HWRP. LWRPs have a file in /resources and a file in /providers.`